client/web/workflow/src/views/Workflow/List.vue

```diff
@@ -164,11 +164,8 @@
             />
           </b-dropdown-item>
 
-          <b-dropdown-item
-            variant="light"
-          >
+          <b-dropdown-item>
             <font-awesome-icon
-              class="text-primary"
               :icon="['fas', 'file-export']"
             />
 
```

Comment:
The export button is still a link so it appears blue. Please add text-dark, same as is for permissions button